copy/copy.go

```diff
@@ -133,6 +133,11 @@ type Options struct {
 	// Invalid when copying a non-multi-architecture image. That will probably
 	// change in the future.
 	EnsureCompressionVariantsExist []OptionCompressionVariant
+
+	// ForceCompressionFormat allows end users to force selected compression format
+	// (set in DestinationCtx.CompressionFormat) which ensures that while copying
+	// blobs of other compression algorithms are not reused.
```

Comment:
Non-blocking, this works fine:

Maybe something like
```suggestion
	// ForceCompressionFormat ensures that the compression algorithm set in
	// DestinationCtx.CompressionFormat is used exclusively, and blobs of other
	// compression algorithms are not reused.
```
would say the same thing a tiny bit more directly.